dev/builddeps-veloxbe.sh

```diff
@@ -25,6 +25,8 @@ ENABLE_EP_CACHE=OFF
 ARROW_ENABLE_CUSTOM_CODEC=OFF
 ENABLE_VCPKG=OFF
 RUN_SETUP_SCRIPT=ON
+VELOX_REPO=https://github.com/oap-project/velox.git
+VELOX_BRANCH=update
```

Comment:
Can we assign an empty default value if user does not special it ? Otherwise after this pr, we may need to change two code place to make sure the default repo/branch.

Reply:
thanks, updated.  
* `./dev/buildbundle-veloxbe.sh` exec, velox_repo/velox_branch/velox_home is defined in `get_velox.sh`
* `./dev/buildbundle-veloxbe.sh --velox_repo=xxx --velox_branch=xxx --velox_home=xxx` exec, velox_repo/velox_branch/velox_home is defined in `builddeps-veloxbe.sh`